actions/create_category_MOD.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  name: 'Create Category Channel',
+  name: 'Create Category',
```

Comment:
use the displayName property instead of changing the mods name, the method you used will broke all previous Rawdata